src/model64.c

```diff
@@ -545,6 +541,25 @@ static int32_t search_anim_index(model64_t *model, const char *name)
     return -1;
 }
 
+static void alloc_anim_slot(model64_t *model, model64_anim_slot_t slot)
+{
+    if(model->active_anims[slot]) {
+        return;
+    }
```

Comment:
In general, this style of coding is slower on N64 because you pay the overhead of a function call (which includes switching to a different instruction cache line) just to check whether there's something to do. It would be better to move this check into all callers (and even mark it as unlikely) to make sure it impacts near zero during runtime